src/IO/H5/Helpers.cpp

```diff
@@ -9,6 +9,7 @@
 #include <string>
 #include <type_traits>
 
+#include "DataStructures/BoostMultiArray.hpp"
```

Comment:
Can you factor the read_data changes into a separate commit please?

Reply:
Just to be clear, you want only the include multiarray line in the other commit and all other changes in this commit?

Reply:
No, I want all the changes you are making to `read_data` in its own commit, not in the Boost multiarray commit and not in the EoS commit.

Reply:
Ok, so to make sure I understand, you want 5 commits in total:

1. The root directory h5::File commit
2. The boost multiarray commit
3. The read_data commit
4. The EoS commit
5. The FileTestDefs commit

Reply:
Almost, you'll want to change the order of the last two commits so each individual commit compiles and passes all checks and tests :)